Modeled after the patch for gcc5

Only for Solaris 11 and Solaris 12, unless Illumos/OI/OmniOS
have the same updated Solaris header files

--- gcc-4.8.5/fixincludes/fixinc.in.orig	2016-12-15 22:23:42.167811239 +0100
+++ gcc-4.8.5/fixincludes/fixinc.in	2016-12-15 22:26:46.452681454 +0100
@@ -471,6 +471,13 @@
 # DOS/Windows filesystems.
 find . -name '[Dd][Oo][Nn][Ee]' -exec rm -f '{}' ';'
 
+# We only want limits.h, syslimits.h, and feature_tests.h fixed on Solaris
+for file in `find . -type f | \
+             egrep -v './(limits.h|syslimits.h|sys/feature_tests.h)'` ; do
+  echo Removing $file
+  rm -f $file
+done
+
 if test $VERBOSE -gt 1
 then echo 'Cleaning up unneeded directories:' ; fi
 cd $LIB
